napari/_vispy/vispy_axes_visual.py

```diff
@@ -68,61 +115,82 @@ def __init__(self, axes, camera, dims, parent=None, order=0):
         self._camera = camera
         self._scale = 1
 
-        # note order is z, y, x
-        self._default_data = np.array(
-            [[0, 0, 0], [0, 0, 1], [0, 0, 0], [0, 1, 0], [0, 0, 0], [1, 0, 0]]
+        # Target axes length in canvas pixels
+        self._target_length = 80
+        # CYMRGB for 6 axes data in x, y, z, ... ordering
+        self._default_color = [
+            [0, 1, 1, 1],
+            [1, 1, 0, 1],
+            [1, 0, 1, 1],
```

Comment:
```suggestion
        # CMYRGB for 6 axes data in x, y, z, ... ordering
        self._default_color = [
            [0, 1, 1, 1],
            [1, 0, 1, 1],
            [1, 1, 0, 1],
```

👆 This order is much more common, I think we should adopt it here.